agent/hcp/telemetry/otel_sink.go

```diff
@@ -118,6 +118,8 @@ func NewOTELSink(opts *OTELSinkOpts) (*OTELSink, error) {
 	}, nil
 }
 
+var _ gometrics.MetricSink = &OTELSink{}
```

Comment:
Why was this added?

Reply:
This was just me trying to figure out which interface this type maps to and implements. Partly documentation and partly (generally) checking that an implementation conforms to an expected interface